### DISCUSSION TITLE
Pricing: if I split the stacks, how many credits do I need?

### DISCUSSION THREAD

**Discussion by nbcchen** [serverless/serverless#12758] [Q&A]
If I use [serverless-plugin-split-stacks](https://www.serverless.com/plugins/serverless-plugin-split-stacks) and split the stack into N nested stacks, with N being the number of Lambda functions, do I get charged with 1 credit per stage per region, or N credits per stage per region?

**Comment by garethmcc** [comment]
@nbcchen This entirely depends on the way the plugin was written. I have tried to get in touch with the author of the plugin to confirm these details with them but have not received a response. It is possible that each individual stack could require a credit, but would need to be confirmed

**Reply by nbcchen** [reply]
Ok. Is there a way to add me as a watcher to your question to the author?

**Reply by nbcchen** [reply]
Any updates?

**Reply by garethmcc** [reply]
@nbcchen I emailed the author directly so there would not be anything to watch. I have not yet received a response
